src/azure-cli/azure/cli/command_modules/acr/connected_acr.py

```diff
@@ -0,0 +1,298 @@
+                             registry_name=registry_name,
+                             connected_registry_name=connected_acr_name,
+                             connected_registry_update_parameters=connected_acr_update_parameters)
+    except ValidationError as e:
+        raise CLIError(e)
+
+
+def acr_connected_acr_delete(cmd,
+                             client,
+                             connected_acr_name,
+                             registry_name,
+                             yes=False,
+                             resource_group_name=None):
+
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+
+    #TODO disable tokens
+    user_confirmation("Are you sure you want to delete the Connected Registry '{}' from '{}'?".format(
+        connected_acr_name, registry_name), yes)
+    try:
+        return client.delete(resource_group_name, registry_name, connected_acr_name)
+    except ValidationError as e:
+        raise CLIError(e)
+
+
+def acr_connected_acr_list(cmd,
+                           client,
+                           registry_name,
+                           parent=None,
+                           cascading=False,
+                           resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+    #TODO Cascading logic and different formattings.
+    return client.list(resource_group_name, registry_name)
+
+
+def acr_connected_acr_show(cmd,
+                           client,
+                           connected_acr_name,
+                           registry_name,
+                           resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+    return client.get(resource_group_name, registry_name, connected_acr_name)
+
+
+def _create_sync_token(cmd,
+                       resource_group_name,
+                       registry_name,
+                       connected_acr_name,
+                       repositories,
+                       mode):
+    token_client = cf_acr_tokens(cmd.cli_ctx)
+
+    mode = mode.lower()
+    if not any(option for option in ConnectedRegistryModes if option.value == mode):
+        raise CLIError("usage error: --mode supports only 'registry' and 'mirror' values.")
+    repository_actions_list = [[repo] + REPO_SCOPES_BY_MODE[mode] for repo in repositories]
+    gateway_actions_list = [[connected_acr_name] + DEFAULT_GATEWAY_SCOPE]
+
+    try:
+        scope_map_id = create_default_scope_map(cmd, resource_group_name, registry_name,
+                                                SYNC_SCOPE_MAP_NAME.format(connected_acr_name),
+                                                repository_actions_list, gateway_actions_list,
+                                                "Created by connected registry: {}".format(connected_acr_name))
```

Comment:
Can you please update the description so customers know this token is related to syncing the connected registry?